lint-staged.config.js

```diff
@@ -1,11 +1,24 @@
 // @ts-check
 
+const { prettierSupportedFileExtensionsByContext } = require('@fluentui/scripts/prettier');
+
+const commands = {
+  format: 'prettier --write',
+  /**
+   * Run eslint in fix mode for applicable files followed by prettier.
+   * The eslint wrapper handles filtering which files should be linted, since we need to both:
+   * - respect ignore files (which eslint doesn't do by default when passed a specific file path)
+   * - match the set of files that are linted by the package's normal `lint` command
+   */
+  lint: 'node ./scripts/lint-staged/eslint',
+};
+
 // https://www.npmjs.com/package/lint-staged
 module.exports = {
-  // Run eslint in fix mode for applicable files followed by prettier.
-  // - The eslint wrapper handles filtering which files should be linted, since we need to both:
-  //   - respect ignore files (which eslint doesn't do by default when passed a specific file path)
-  //   - match the set of files that are linted by the package's normal `lint` command
-  // - Prettier must be run in sequence after eslint since both of them can modify files
-  '*': ['node ./scripts/lint-staged/eslint', 'prettier --write'],
+  [`**/*.{${
+    (prettierSupportedFileExtensionsByContext.stylesheets,
+    prettierSupportedFileExtensionsByContext.markdown,
+    prettierSupportedFileExtensionsByContext.others)
```

Comment:
Is this missing a function call to merge the arrays? Running something similar in the browser console it only returns the last comma operand. ``` `${(['a','b'],['c','d'])}` ``` returns `c,d`

Reply:
Went ahead and fixed this since @Hotell is out and the PR is ready to go otherwise